examples/SonDisk_Proc3/SerialConnectorInput_D.pde

```diff
@@ -0,0 +1,34 @@
+
+void serialEvent (Serial myPort) {
+   String theInput = myPort.readStringUntil('\n'); //Gets val
+   if (processInputs(theInput)) displayInputs();
+}
+
+
+boolean processInputs(String tInputs) {
+   if (tInputs.charAt(0) == 'S') {
+      // print(tInputs);
+     String theValues[] = split(tInputs,",");
+     if(theValues.length!=10) return false; // exit if the expected number of variables not present
+     sonic0 = int(theValues[1]);
+     sonic1 = int(theValues[2]);
+     sonic2 = int(theValues[3]);
+     sonic3 = int(theValues[4]);
+     sonic4 = int(theValues[5]);
+     sonic5 = int(theValues[6]);
+     sonic6 = int(theValues[7]);
+     sonic7 = int(theValues[8]);
+   }
+return true;
```

Comment:
Can you run some auto-indentation over the file?

Reply:
used CTRL-T on each tab